airflow/ui/src/pages/DagsList/Dag/Graph/TaskNode.tsx

```diff
@@ -34,7 +34,9 @@ export const TaskNode = ({
     isMapped,
     isOpen,
     label,
+    operator,
     setupTeardownType,
+    type: nodeType,
```

Comment:
Oh yeah, maybe we should change that.

Reply:
Yes. I tried to quickly do that and it wasn't super straigh forward because for now types and node conversion on the front-end is a little intricated. (`node_type` => `nodeType`, we already do that for is_setup_teardown, but I needed more time to make it work and didn't want to spend to long on it at the moment)

Maybe doing the planned refactor / simplification of that part of the code could help first.